pkg/rotator/rotator.go

```
@@ -692,6 +706,19 @@ func (r *ReconcileWH) Reconcile(ctx context.Context, request reconcile.Request)
 	}
 
 	if secret.GetDeletionTimestamp().IsZero() {
+		if r.refreshCertIfNeededDelegate != nil {
+			rotatedCA, err := r.refreshCertIfNeededDelegate()
+			if err != nil {
+				crLog.Error(err, "error rotating certs on secret reconcile")
+				return reconcile.Result{}, nil
```

Why are we swallowing the error here?

---

good catch! returning err now.